ast/misc.go

```diff
@@ -109,7 +109,6 @@ type DeallocateStmt struct {
 	stmtNode
 
 	Name string
-	ID   uint32 // For binary protocol, there is no Name but only ID.
```

Comment:
Because `ast.DeallocateStmt` is not used when we call `Session.DropPreparedStmt` directly, it is only used in sql query like `deallocate 'abc'` statement, where there is no ID.